dotnet/src/SemanticKernel/Planning/SKContextPlanningExtensions.cs

```diff
@@ -9,8 +9,10 @@
 using Microsoft.SemanticKernel.SkillDefinition;
 using static Microsoft.SemanticKernel.CoreSkills.PlannerSkill;
 
+#pragma warning disable IDE0130
 // ReSharper disable once CheckNamespace // Extension methods
 namespace Microsoft.SemanticKernel.Orchestration;
+#pragma warning restore IDE0130
```

Comment:
Can you change the namespace for this one to `SemanticKernel.UnitTests.Planning` and remove the pragmas and ReSharper disable comments

Reply:
No, we explicitly want to keep this in the same orchestration namespace to reduce what devs are required to import.

Reply:
((This was requested of me in previous reviews, just now adding the IDE pragma so I don't have to keep fighting it))